src/environment.py

```diff
@@ -274,8 +275,8 @@ def generate(config: dict, app_name: str, site_url: str, db_uri: str) -> dict:
     # Update env with provided extra_env
     try:
         env.update(json.loads(config["extra_env"]))
-    except json.JSONDecodeError:
-        raise json.jSONDecodeError("extra_env is not valid JSON")
+    except json.JSONDecodeError as excpt:
+        raise json.JSONDecodeError("extra_env is not valid JSON", excpt.doc, excpt.pos)
```

Comment:
I think this is a typo "excpt"?

Reply:
No ?  
What would you choose as variable name ?

Reply:
After discussion, I changed the name to "exc"